.github/workflows/judoscale-delayed_job-test.yml

```diff
@@ -14,15 +14,20 @@ jobs:
       matrix:
         gemfile:
           - Gemfile
+          - Gemfile-activerecord-7-0
```

Comment:
Adding AR v7 tests so we run against all stables currently maintained: 6.1, 7.0, 7.1